packages/compass-e2e-tests/helpers/commands/select-connection-menu-item.ts

```diff
@@ -11,7 +11,7 @@ export async function selectConnectionMenuItem(
   await browser.$(selector).waitForDisplayed();
 
   // workaround for weirdness in the ItemActionControls menu opener icon
-  await browser.clickVisible(Selectors.ConnectionsTitle);
+  await browser.clickVisible(Selectors.SidebarNewConnectionButton);
```

Comment:
This has a side-effect that wasn't there before. Clicking the ConnectionsTitle did nothing other than move focus somewhere else. (was to give hover another chance of maybe doing something on windows, if I remember correctly.)

Clicking new connection clears the connect form. Maybe nothing that calls this command cares. I don't know.

Reply:
Gotcha, makes sense. I'll make it click something that isn't interactive 👍

Reply:
Is this solving the same problem that https://github.com/mongodb-js/compass/pull/3660/files#diff-93fd67837e378b3f719a92f9fde522278291f129ab8fc56ace9b36d514cbba0aR19 is aiming to solve?